README.md

```diff
@@ -43,3 +44,103 @@ npm run dev
+- Before running any Docker commands, ensure you are logged into GHCR using your GitHub personal access token (PAT):
+
+```bash
+docker login ghcr.io -u <your_github_username> -p <your_PAT>
+```
+
+- Ensure you have Containerd installed and running
+- Ensure you have Docker Buildx installed and configured
+- Ensure you are using the builder instance you create using steps 1-4 below
+- Ensure you have Docker installed and running
+**Image Naming Convention:**  
+The Docker images follow the naming convention:  
+`ghcr.io/refactor-group/refactor-platform-fe/<branch>:latest`  
+
+Where:
+
+- `refactor-group` is your GitHub organization.
+- `refactor-platform-fe` is the repository name.  
+- `<branch>` is the name of the branch, with underscores converted to dashes.
+- `latest` is the tag.
+
+**Useful Commands:**
+
+```bash
+# Docker Buildx: Enhanced Image Management
+
+# 1. Inspect Docker Buildx
+docker buildx version # Verify Docker Buildx is installed
+
+# 2. Create a new builder instance (if needed)
+docker buildx create --name mybuilder --driver docker-container # Creates a builder instance named 'mybuilder' using the docker-container driver
+
+# 3. Use the builder
+docker buildx use mybuilder # Sets 'mybuilder' as the current builder
+
+# 4. Inspect the builder
+docker buildx inspect --bootstrap # Displays details and ensures the builder is running
+
+# 5. Login to GHCR
+docker login ghcr.io -u <your_github_username> -p <your_PAT> # Authenticates with GitHub Container Registry using your username and PAT
+
+# 6. Build the image for multiple architectures and push to registry
+docker buildx build --platform linux/amd64,linux/arm64 -t ghcr.io/refactor-group/refactor-platform-fe:<branch> --push . # Builds for specified architectures and pushes to GHCR
+
+# 7. Build the image for multiple architectures and load to local docker daemon
+docker buildx build --platform linux/amd64,linux/arm64 -t ghcr.io/refactor-group/refactor-platform-fe:<branch> --load . # Builds for specified architectures and loads to local docker daemon
+
+# 8. Tag the image
+docker tag ghcr.io/refactor-group/refactor-platform-fe:<branch> ghcr.io/refactor-group/refactor-platform-fe:<branch>:latest # Creates an additional tag 'latest' for the image
+
+# 9. Push the image
+docker push ghcr.io/refactor-group/refactor-platform-fe:<branch>:latest # Uploads the image to the container registry
+
+# 10. Pull the image
+docker pull ghcr.io/refactor-group/refactor-platform-fe:<branch>:latest # Downloads the image from the container registry
+
+# 11. Run the image
+docker run -p 3000:3000 ghcr.io/refactor-group/refactor-platform-fe:<branch>:latest # Starts a container from the image, mapping port 3000
+
+# 12. Inspect the image
+docker inspect ghcr.io/refactor-group/refactor-platform-fe:<branch>:latest # Shows detailed information about the image
+
+# 13. Remove the image
+docker rmi ghcr.io/refactor-group/refactor-platform-fe:<branch>:latest # Deletes the image from the local machine
+```
```

Comment:
What do you think about moving these into a new directory `docs/runbooks` and in a file just like you have in the backend called `Container-README.md`? These are very helpful to have, but I don't think they belong in the project's root `README.md`. Feel free to link to this new `Container-README.md` here though - that is helpful to surface in the root `README.md`.